docs/source/building/environment-vars.rst

```diff
@@ -20,59 +20,61 @@ override this behavior.)
       64-bit.  The value depends on the ``ARCH`` environment variable, and
       defaults to the architecture the interpreter running conda was
       compiled with
+  * - ``CMAKE_GENERATOR``
+    - The CMake generator string for the current build environment. On Unix
+      systems, this is always "Unix Makefiles". On Windows, it is generated
+      according to the Visual Studio version activated at build time,
+      e.g. "Visual Studio 9 2008 Win64"
   * - ``CONDA_BUILD=1``
     - Always set
-  * - ``SRC_DIR``
-    - Path to where source is unpacked (or cloned). If the source file is not
-      a recognized file type (right now, ``.zip``, ``.tar``, ``.tar.bz2``,
-      ``.tar.xz``, and ``.tar``), this is a directory containing a copy of the
-      source file
+  * - ``CPU_COUNT``
+    - Number of CPUs on the system, as reported by
+      ``multiprocessing.cpu_count()``
+  * - ``DIRTY``
+    - Set to 1 if `--dirty` flag passed to `conda build` command.  May be used to skip parts of
+      build script conditionally (downloads, extraction, other things that need not be repeated
+      for faster iteration time when developing recipes)
+  * - ``HTTP_PROXY``
+    - Inherited from your shell environment.
+  * - ``HTTPS_PROXY``
+    - Inherited from your shell environment.
+  * - ``LANG``
+    - Inherited from your shell environment.
+  * - ``MAKEFLAGS``
+    - Inherited from your shell environment. May be used to set additional
+      arguments to make, such as `-j2`, which will use 2 CPU cores to build
+      your recipe.
+  * - ``NPY_VER``
+    - Numpy version building against (Set via `--numpy` arg or via `CONDA_NPY` environment variable.)
+  * - ``PATH``
+    - Inherited from your shell environment, and augmented with ``$PREFIX/bin``
```

Comment:
I had a merge conflict here.  Please check and make sure this line reads the way you want.